beamer/models/request.py

```diff
@@ -49,7 +49,12 @@ def __init__(
 
     fill = pending.to(filled) | filled.to(filled) | ignored.to(filled) | claimed.to(claimed)
     try_to_fill = pending.to(filled)
-    try_to_claim = filled.to(claimed) | pending.to(claimed) | claimed.to(claimed)
+    try_to_claim = (
+        filled.to(claimed)
+        | pending.to(claimed)
+        | claimed.to(claimed)
+        | l1_resolved.to(l1_resolved)
```

Comment:
this can happen when someone challenges the agent's claim after it was already l1_resolved